src/ui/public/courier/data_source/build_query/from_kuery.js

```diff
@@ -1,8 +1,8 @@
 import _ from 'lodash';
 import { fromKueryExpression, toElasticsearchQuery, nodeTypes } from '../../../kuery';
 
-export function buildQueryFromKuery(indexPattern, queries) {
-  const queryASTs = _.map(queries, query => fromKueryExpression(query.query));
+export function buildQueryFromKuery(indexPattern, queries, useSimple = false) {
+  const queryASTs = _.map(queries, query => fromKueryExpression(query.query, undefined, useSimple));
```

Comment:
I think a separate `from_simple_kuery.js` module might be more clear than passing around a boolean flag, even if it duplicates a little code.